README.md

```diff
@@ -1,13 +1,40 @@
 # mal-effect-calculations
-
 This repository contains scripts for the impact assessment of the MAL Plan.
 Learn more about the MAL Plan here: https://www.hsl.fi/hsl/mal
 
-## How to run
+Scripting language is mainly R so far. R enables easy plotting and manipulating of spatial features.
 
+## How to run
 1. Clone this repository.
-1. In RStudio, select `File > Open Project... > mal-effect-calculations.Rproj`.
-1. Run `renv::restore()`. Now, you are using packages listed in `renv.lock`.
+2. In RStudio, select `File > Open Project... > mal-effect-calculations.Rproj`.
+3. Set correct environment for `config.yml` file. 
+4. Run `renv::restore()`. Now, you are using packages listed in `renv.lock`.
+
+## Data management
+Raw data is stored in HSL Sharepoint and needs to be copied to local `/data` folder.
+Scripts have relative paths to data directory and sub-folders.
+
+Results from scripts and analysis-friendly tidy data is kept saved to results sub-folder.
+
+Non relative data paths are set `config.yml` file. 
+These include for example Helmet-model-system data and result files.
+
+Config file environments are used to distinguish for example different MAL planning versions or other projects. 
+Environment is set via `Sys.setenv(R_CONFIG_ACTIVE = "V2030_ve0_rm")` command. 
+
+Pipeline example for data processing is stored in `R/run.R`.
+
+## Version control
+We using R-package `renv` that locks versions of packages. When doing analysis first time you should
+load packages with command: `renv::restore(repos = "https://cloud.r-project.org", clean = TRUE)`.
+If packages are already loaded, use command `renv::restore()`.
+
+R-version is `4.0.2`. This is not changed by `renv`, so you might want to check R/RStudio Global Options.
```

Comment:
I'm using R 4.0.4. This will have an effect on `renv` merge conflict. I'm inclined to continue using 4.0.4 as it was commited to his package earlier. 😕

Reply:
Yep, my mistake as I didn't notice the version you had used. I'll fix this.